packages/ui/Resizable/useResizable.js

```diff
@@ -0,0 +1,50 @@
+import {useCallback, useContext, useEffect, useRef} from 'react';
+
+import ResizeContext from './ResizeContext';
+
+/**
+ * A custom hook which returns handler that indicates a resize is necessary
+ * @function useResizable
+ * @memberof ui/Resizable
+ * @ui
+ */
+const useResizable = (props, config) => {
+	// `resize` is the name of the event on the component to listen for size changes.
+	const {resize} = config;
```

Comment:
Where is `filter` from `Resizable` config?

Reply:
I think I missed it. I added the logic of the filter.